docs/src/ComboBox.doc.js

```diff
@@ -0,0 +1,732 @@
+    />
+    <MainSection.Subsection
+      title="Keyboard interaction"
+      description={`
+    * Hitting \`Enter\` or \`Space\` key on the ComboBox's trigger opens the options list
+    * \`Escape\` key closes the options list, while moving focus back on the ComboBox's trigger
+    * Arrow keys are used to navigate items within the options list
+    * \`Enter\` key selects an item within the options list
+    * \`Tab\` or \` Shift + Tab\` close the options list and move focus accordingly
+  `}
+    />
+  </MainSection>,
+);
+
+card(
+  <MainSection
+    name="Localization"
+    description={`Be sure to localize the \`helperText\`, \`errorMessage\`, \`noResultText\`, \`label\`, \`placeholder\`, \`accessibilityShowButtonLabel\`,  and \`accessibilityClearButtonLabel\` props. \`options\` and \`value\` should be localized for those cases that can be translated. Note that localization can lengthen text by 20 to 30 percent.`}
+  >
+    <MainSection.Card
+      cardSize="lg"
+      defaultCode={`
+function ComboBoxExample(props) {
+  const PRONOUNS = [
+    'ell@ / l@ / -@',
+    'ella / la / le / -a',
+    'elle / le / -e',
+    'ellx / lx / -x',
+    'él / lo / le / -o',
+  ];
+
+  const options = PRONOUNS.map((pronoun, index) => ({ label: pronoun, value: 'value'+index }));
+
+  return (
+    <ComboBox
+      accessibilityClearButtonLabel="Remueve la lista de pronombres seleccionados"
+      accessibilityShowButtonLabel="Muestra la lista de pronombres para este campo"
+      label="Pronombres"
+      id="localization"
+      noResultText="No se encontró ninguna coincidencia"
+      options={options}
+      placeholder="Añade tus pronombres"
+      helperText="Elige hasta 2 grupos de pronombres para que aparezcan en tu perfil y otras personas sepan cómo referirse a ti. Puedes editarlos o eliminarlos en cualquier momento."
+    />
+  );
+}
+`}
+    />
+  </MainSection>,
+);
+
+card(
+  <MainSection name="Variants">
+    <MainSection.Subsection
+      description="ComboBox can be used as a controlled or an uncontrolled component. An uncontrolled ComboBox stores its own state internally and update it based on the user input. On the other side, a controlled ComboBox's state is managed by a parent component. The parent component's state passes new values through props to the controlled component which notifies changes through event callbacks."
+      title="Controlled vs Uncontrolled"
+    >
+      <MainSection.Card
+        cardSize="lg"
+        title="Uncontrolled ComboBox"
+        description={`An uncontrolled ComboBox should be used for basic cases where no default value or tags are required. By passing \`options\` values to ComboBox, the component takes control and has fully manages its internal state. Don't use the \`inputValue\` or \`selectedOptions\` props, any value different from \`null\` and \`undefined\` activates a controlled ComboBox.`}
```

Comment:
"... and fully manages its internal state." 

I also don't quite understand the last sentence so it might need to be rephrased a bit?

Reply:
DONE